pkg/batch/executor_test.go

```diff
@@ -24,12 +68,13 @@ func testReadAfterWrite(t *testing.T) {
 	// 3. writer (w1) returns (Current version: v1)
 	// 4. reader (r2) starts
 	// 5. both readers (r1,r2) return with v1 as their response.
-	var db = sync.Map{}
-	db.Store("v", "v0")
+	var db = &db{sync.Map{}, 0}
```

Comment:
I think these are the defaults; you could just use

```suggestion
	var db = &db{}
```